configuration/concepts/adult_hiv_initial_form.csv

```diff
@@ -0,0 +1,97 @@
+uuid,Members,Answers,Fully specified name:fr,Fully specified name:en,Short name:en,Short name:fr,Data class,Data type,_version:1,_order:1000
+5f38b80a-d4e2-4623-bb15-6662c33ab4f9,,,Nombre d'enfant en charge,Number of children in charge,,,Misc,Numeric,,
+6221e521-79c9-4a28-b3e5-e96df371b855,,,Age du dernier enfant,Age of last child,,,Misc,Numeric,,
+b2031349-b60a-433d-880a-8668cc040d2f,,,Hétérosexuel,Heterosexual,,,Misc,N/A,,
+9801e8cd-da10-4c52-adee-27cd6939adbb,,,Bisexuel,Bisexual,,,Misc,N/A,,
+c540d003-cdb5-434c-baa4-0b2b3e5e3205,,,Homosexuel,Homosexual,,,Misc,N/A,,
+211f0857-61a3-4049-9777-374c4a592453,,b2031349-b60a-433d-880a-8668cc040d2f;9801e8cd-da10-4c52-adee-27cd6939adbb;c540d003-cdb5-434c-baa4-0b2b3e5e3205,Orientation sexuelle,Sexual orientation ,,,Misc,Coded,,
+b7648479-e480-4772-b646-a5abc07bc4f7,,a2065636-5326-40f5-aed6-0cc2cca81ccc;b497171e-0410-4d8d-bbd4-7e1a8f8b504e,Partenaire informé,Informed partner,,,Misc,Coded,,
+7acfafa4-f19b-485e-97a7-c9e002dbe37a,,,Positif,Positive,,,Misc,N/A,,
+718b4589-2a11-4355-b8dc-aa668a93e098,,,Négatif,Negative,,,Misc,N/A,,
+b09900a6-4646-4804-b460-63be72879a13,,,Ne sait pas,Do not know,,,Misc,N/A,,
+351d2ae2-597e-471d-859c-ac0f782e06ec,,Positive;Negative;Do Not Know,Statut Partenaire,Status Partner,,,Misc,Coded,,

```

Comment:
@lauravignoli Could you please double check the above concept? Noticed the following issues:
`Fully specified name:fr` = `Positive;Negative` (the text is in English)
The corresponding English translation (`Do Not Know`) does not look right.
And also the short name seems to be unrelated to the fully specified name.

Reply:
@tmvumbi2 This is working as expected to me? The possible answers are Positive, Negative or Do Not Know.

![statusPartner](https://user-images.githubusercontent.com/1198827/54992518-2c7db480-4fc8-11e9-8356-a3b0da0f6839.png)

Reply:
@lauravignoli oh you right, I didn't see that this was an answer list